cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -11,43 +13,38 @@
 import cfme.intelligence.chargeback.rates as rates
 from cfme import test_requirements
 from cfme.base.credential import Credential
-from cfme.cloud.provider.azure import AzureProvider
+from cfme.cloud.provider import CloudProvider
+from cfme.cloud.provider.ec2 import EC2Provider
 from cfme.cloud.provider.gce import GCEProvider
 from cfme.common.provider import BaseProvider
+from cfme.common.provider import CloudInfraProvider
 from cfme.common.vm import VM
-from cfme.infrastructure.provider.rhevm import RHEVMProvider
-from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.intelligence.reports.reports import CustomReport
 from cfme.utils.blockers import BZ
 from cfme.utils.log import logger
 from cfme.utils.wait import wait_for
-from fixtures.provider import setup_or_skip
 
 pytestmark = [
     pytest.mark.tier(2),
-    pytest.mark.meta(blockers=[BZ(1531600, forced_streams=["5.9"]),
-                               BZ(1511099, forced_streams=["5.7", "5.8"],
+    pytest.mark.meta(blockers=[BZ(1511099, forced_streams=["5.9"],
                                   unblock=lambda provider: not provider.one_of(GCEProvider)),
-                               BZ(1531554, forced_streams=["5.8"])]),
-    pytest.mark.provider([VMwareProvider, RHEVMProvider, AzureProvider, GCEProvider],
-                         scope='module',
+                               ]),
+    pytest.mark.provider([CloudInfraProvider], scope='module',
                          required_fields=[(['cap_and_util', 'test_chargeback'], True)]),
     test_requirements.chargeback,
 ]
 
+# Allowed deviation between the reported value in the Chargeback report and the estimated value.
+DEVIATION = 1
+
 
 @pytest.yield_fixture(scope="module")
-def clean_setup_provider(request, provider):
-    BaseProvider.clear_providers()
-    setup_or_skip(request, provider)
+def clean_setup_provider(request, has_no_providers_modscope, setup_provider_modscope,
```

Comment:
Requited: this can be removed and fixtures can be added to pytestmark:
pytest.mark.usefixtures('has_no_providers_modscope', 'setup_provider_modscope')